app/views/layouts/_user_config.html.erb

```diff
@@ -3,139 +3,13 @@
     var bodyClass = localStorage.getItem('config_body_class');
     document.body.className = bodyClass;
     if (bodyClass.includes('night-theme')) {
-      document.getElementById('body-styles').innerHTML = '<style>\
-        :root {\
-        --theme-background: #0d1219;\
-        --theme-color: #fff;\
-        --theme-opposite-color: #0d1219;\
-        --theme-logo-background: #0a0a0a;\
-        --theme-logo-color: #fff;\
-        --theme-code-background: #29292e;\
-        --theme-code-color: #e6db74;\
-        --theme-reaction-background: #202c3d;\
-        --theme-anchor-color: #17a1f6;\
-        --theme-secondary-color: #cedae2;\
-        --theme-secondary-color-border: 1px solid #1c283b;\
-        --theme-top-bar-background: #1c2938;\
-        --theme-top-bar-background-hover: #27384c;\
-        --theme-top-bar-color: #fff;\
-        --theme-top-bar-search-background: #424a54;\
-        --theme-top-bar-search-color: #fff;\
-        --theme-top-bar-write-background: #00af81;\
-        --theme-top-bar-write-color: #fff;\
-        --theme-container-background: #141f2d;\
-        --theme-container-accent-background: #202c3d;\
-        --theme-container-background-hover: #1c2c3f;\
-        --theme-gradient-background: linear-gradient(to right, #293d56 8%, #282833 18%, #293d56 33%);\
-        --theme-container-color: #fff;\
-        --theme-container-box-shadow: none;\
-        --theme-container-border: 1px solid #22303f;\
-        --theme-subtle-border: 1px solid #1f2c3a;\
-        --theme-social-icon-invert: invert(100);\
-        --theme-social-icon-color: #cedae2;\
-        --theme-invert-image: invert();\
-        --theme-current-article-background: #1143ca;\
-        --theme-series-article-background: #314d9c;\
-        --theme-series-article-hover: #1b2e63;</style>'
+      document.getElementById('body-styles').innerHTML = '<style><%= Rails.application.assets["themes/night.css"].to_s.squish.html_safe %></style>';
```

Comment:
I move inline styling for each theme from erb file to it's own SCSS file and instead I reference it here.

Reply:
I'm just going to pull your PR locally to dynamically load the stylesheet instead of adding it as inner HTML.